### ISSUE TITLE
Hope add a click animation(Material Design Wave)

### ISSUE THREAD

**Issue by RA1NO3O** [pedromassango/bottom_navy_bar#70]

**Comment by pedromassango** [comment]
Hi @RA1NO3O 
Do you have any link to learn more about the animation you are asking?

**Comment by RA1NO3O** [comment]
Yes and I just hope you can build an attribute to enable this :)

**Comment by RA1NO3O** [comment]
> Hi @RA1NO3O 
> Do you have any link to learn more about the animation you are asking?
> 

Maybe check this out:
https://www.kikt.top/posts/flutter/flutter-click-event-1/

**Comment by pedromassango** [comment]
Is that what you want?
I fear this will not be possible because it may conflict with the selected item color and animation.

![vvg](https://user-images.githubusercontent.com/33294549/98649055-63d4dc80-2337-11eb-989a-e6365e4487bf.jpg)

Maybe I misunderstood you, can you help me understand how it should exactly work? :)

**Comment by RA1NO3O** [comment]
Just trigger this animation when pressing, like almost android native button did.

**Comment by RA1NO3O** [comment]
https://flutter.dev/docs/cookbook/gestures/ripples

**Comment by pedromassango** [comment]
Will get to this when I have time

**Comment by antonio-pedro99** [comment]
@pedromassango and the time has never came xD
